docs/schema/v0/targeting.json

```diff
@@ -0,0 +1,558 @@
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "+": {
+          "title": "Addition Operation",
+          "description": "Addition; associative, will accept and unlimited amount of arguments.",
+          "$ref": "#/$defs/associativeOp"
+        },
+        "*": {
+          "title": "Multiplication Operation",
+          "description": "Multiplication; associative, will accept and unlimited amount of arguments.",
+          "$ref": "#/$defs/associativeOp"
+        }
+      }
+    },
+    "unaryOp": {
+      "type": "array",
+      "minItems": 1,
+      "maxItems": 1,
+      "items": {
+        "$ref": "#/$defs/args"
+      }
+    },
+    "unaryRule": {
+      "title": "Unary Operation",
+      "description": "Any primitive JSONLogic operation with 1 operands.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "!": {
+          "title": "Negation Operation",
+          "description": "Logical negation (“not”). Takes just one argument.",
+          "$ref": "#/$defs/unaryOp"
+        },
+        "!!": {
+          "title": "Double Negation Operation",
+          "description": "Double negation, or 'cast to a boolean'. Takes a single argument.",
+          "$ref": "#/$defs/unaryOp"
+        },
+        "max": {
+          "title": "Maximum Operation",
+          "description": "Return the maximum from a list of values.",
+          "$ref": "#/$defs/unaryOp"
+        },
+        "min": {
+          "title": "Minimum Operation",
+          "description": "Return the minimum from a list of values.",
+          "$ref": "#/$defs/unaryOp"
+        }
+      }
+    },
+    "variadicOp": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/$defs/args"
+      }
+    },
+    "variadicRule": {
+      "$comment": "note < and <= can be used with up to 3 ops (between)",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "or": {
+          "title": "Or Operation",
+          "description": "Simple boolean test, with 1 or more arguments. At a more sophisticated level, \"or\" returns the first truthy argument, or the last argument.",
+          "$ref": "#/$defs/variadicOp"
+        },
+        "and": {
+          "title": "",
+          "description": "Simple boolean test, with 1 or more arguments. At a more sophisticated level, \"and\" returns the first falsy argument, or the last argument.",
+          "$ref": "#/$defs/variadicOp"
+        },
+        "<": {
+          "title": "Less-Than/Between Operation. Can be used to test that one value is between two others.",
+          "description": "",
+          "$ref": "#/$defs/variadicOp"
+        },
+        "<=": {
+          "title": "Less-Than-Or-Equal-To/Between Operation. Can be used to test that one value is between two others.",
+          "description": "",
+          "$ref": "#/$defs/variadicOp"
+        },
+        "-": {
```

Comment:
Why don't we have a `/` division operator? 🤔 since we have also `+`, `*` and even `%`, I'd expect `/`

Reply:
I missed that somehow!